src/main/java/org/intermine/biovalidator/CommandLineRunner.java

```diff
@@ -137,30 +205,38 @@ private static void displayMessages(String messageType, List<Message> messages)
 
         @CommandLine.Option(
                 names = {"-t", "--type"},
-                description = "ValidatorType, fasta is used as default if not provided"
-                        + " possible values(fasta, fasta-dna, fasta-rna, fasta-protein)",
+                description = "ValidatorType, "
+                        + "possible values:\n fasta,\n fasta-dna,\n fasta-rna,\n fasta-protein,\n"
+                        + " gff3,\n csv",
```

Comment:
Should tab / tsv be on this list?

Reply:
should I add tsv? I thought CSV and TSV are the same types of file!

Reply:
yes, you should add TSV and TAB.

TSV is a tab separated file. TAB is a tab separated file. CSV is a comma separated file. They are different types of files.

The code may treat them similarly, but this is a text message for the end users. In this case, users will not consider tab and comma separated files the same types of files.

Does that make sense?

Reply:
yes:)